rango/views.py

```diff
@@ -134,41 +109,35 @@ def add_page(request, category_name_slug):
 
         else:
             print(form.errors)
-
     context_dict = {"form": form, "category": category}
     return render(request, "rango/add_page.html", context=context_dict)
 
 
 @login_required
 def restricted(request):
+    """The method render restricted page, if user registered."""
     return render(request, "rango/restricted.html")
 
 
-# A helper method
-def get_server_side_cookie(request, cookie, default_val=None):
-    val = request.session.get(cookie)
+def get_server_side_cookie(request, cookie: str, default_val=None) -> int:
```

Comment:
If it is a help method and used just inside another method you should provide the "_" at the beginning of the func name - to show that this func is just for the local use (no use in other scripts).
`def _get_server_side_cookie...`